python/mxnet/gluon/nn/basic_layers.py

```diff
@@ -416,6 +416,48 @@ def __repr__(self):
                         alpha=self._alpha)
 
 
+class PReLU(HybridBlock):
```

Comment:
what's the difference between this and leakyrelu? Should it be an option of leakyrelu?

Reply:
The difference is leaky relu takes a constant, whereas prelu creates a learnable parameter.

Reply:
Is RReLU a standard name? Could you add reference to the paper?

Reply:
I remember there are a bunch of relus with similar names.

Reply:
Added. Yeah there are a bunch, but I think PReLU is a fairly common one. I know it's built in to PyTorch, Keras for example. I'm not familiar with RReLU.